packages/account/src/Sections/Security/ClosingAccount/closing-account-reason-form.tsx

```diff
@@ -0,0 +1,253 @@
+import React from 'react';
+import { Field, Formik, FormikErrors, FormikValues } from 'formik';
+import { Checkbox, FormSubmitButton, Input, Text } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+
+type TFormValues = {
+    'financial-priorities': boolean;
+    'stop-trading': boolean;
+    'not-interested': boolean;
+    'another-website': boolean;
+    'not-user-friendly': boolean;
+    'difficult-transactions': boolean;
+    'lack-of-features': boolean;
+    'unsatisfactory-service': boolean;
+    'other-reasons': boolean;
+    other_trading_platforms: string;
+    do_to_improve: string;
+};
+const initial_form: TFormValues = {
+    'financial-priorities': false,
+    'stop-trading': false,
+    'not-interested': false,
+    'another-website': false,
+    'not-user-friendly': false,
+    'difficult-transactions': false,
+    'lack-of-features': false,
+    'unsatisfactory-service': false,
+    'other-reasons': false,
+    other_trading_platforms: '',
+    do_to_improve: '',
+};
+type TClosingAccountReasonFormProps = {
+    validateFields: (values: FormikValues) => FormikErrors<FormikValues>;
+    onSubmit: (values: FormikValues) => void;
+    is_checkbox_disabled: boolean;
+    onChangeCheckbox: (
+        values: FormikValues,
+        field_name: keyof TFormValues,
+        setFieldValue: (name: string, values: string | boolean) => void
+    ) => void;
+    character_limit_no: number;
+    onInputChange: (
+        e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+        old_value: string,
+        onChange: (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => void
+    ) => void;
+    onInputPaste: (e: React.ClipboardEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+    remaining_characters: number;
+    onBackClick: () => void;
```

Comment:
Can we rename this props to `onCancel`, as it's only been used with submit button as `onCancel`